backend/routes/stripe.js

```
@@ -25,40 +25,45 @@ const allowedTracks = [
+        metadata,
+      });
+      // Log session in Supabase payment_logs
+      try {
+        await supabase.from('payment_logs').insert([
+          {
+            user_id,
+            amount: session.amount_total ? session.amount_total / 100 : null, // Stripe returns amount in cents
+            currency: session.currency || 'usd',
+            payment_method:
+              (session.payment_method_types &&
+                session.payment_method_types[0]) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)